ndx-simulation-output/ndx-meta.yaml

```diff
@@ -0,0 +1,7 @@
+name: my_simulation_output
```

Comment:
please change name to: ndx-simulation-output

Reply:
done